packages/web-components/src/image/image.stories.ts

```diff
@@ -0,0 +1,249 @@
+        },
+      },
+    },
+    block: {
+      description:
+        'An image can use the argument ‘block’ so that it’s width will expand to fiill the available container space.',
+      table: {
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    border: {
+      description: 'Border surrounding image',
+      table: {
+        type: {
+          summary: 'Use this option to provide minimal visual separation between image and surrounding content.',
+        },
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    borderRadius: {
+      description: 'Optional border radius',
+      table: {
+        defaultValue: {
+          summary: 'small',
+        },
+      },
+      options: Object.values(ImageBorderRadius),
+      control: 'select',
+    },
+    fit: {
+      description: 'Determines how the image will be scaled and positioned within its parent container.',
+      table: {
+        defaultValue: {
+          summary: 'default',
+        },
+      },
+      options: Object.values(ImageFit),
+      control: 'select',
+    },
+    margin: {
+      description: 'Optional 16px margin -- to be supplied by component consumer',
+    },
+    role: {
+      description: 'Aria role -- to be supplied by component consumer',
+      table: {
+        type: {
+          summary:
+            'If images are solely decorative and don’t provide useful information or context, use role=”presentation” to hide them from assistive technologies.',
+        },
+      },
+    },
+    shadow: {
+      description: 'Apply an optional box shadow to further separate the image from the background.',
+      table: {
+        type: {
+          summary:
+            'To give an image additional prominence, use the shadow prop to make it appear elevated. Too many shadows can cause a busy layout, so use them sparingly.',
+        },
+        defaultValue: {
+          summary: false,
+        },
+      },
+    },
+    shape: {
+      description: 'Image shape',
+      table: {
+        defaultValue: {
+          summary: 'square',
+        },
+      },
+      options: Object.values(ImageShape),
+      control: 'select',
+    },
+    src: {
+      description: 'Image source -- to be supplied by component consumer',
+      table: {
+        type: {
+          summary: 'Required',
+        },
+      },
+    },
+  },
+} as ImageStoryMeta;
+
+export const Image = renderComponent(imageTemplate).bind({});
+
+// Block layout
+const imageLayoutBlock = html<ImageStoryArgs>`
+  <div style="border: 1px dotted #43ED35;">
+    <fluent-image block border="true">
+      <img role="presentation" src="https://via.placeholder.com/958x20/ddd.png" />
+      <img role="presentation" src="https://via.placeholder.com/100x100/ddd.png" />
```

Comment:
Is it an expected use case to pass multiple `<img>` elements into the `fluent-image`?

Reply:
No. I was simply showing how different images would respond to attribute selection. I can reduce the first example to just one img.

Reply:
@miroslavstastny You had written about default attribute values in my Divider PR. I can apply the same adjustments here -- where the styles handle the default state (ie. no additional attributes on the DOM), the related attributes do not need to be in the story or component options.ts. For example:

`
/**
 * Image fit
 * @public
 */
export const ImageFit = {
  none: 'none',
  center: 'center',
  contain: 'contain',
  cover: 'cover',
  default: 'default', <--- I can REMOVE this as it is the default
} as const;
/**
 * Types for image fit
 * @public
 */
export type ImageFit = ValuesOf<typeof ImageFit>;

/**
 * Image shape
 * @public
 */
export const ImageShape = {
  circular: 'circular',
  rounded: 'rounded',
  square: 'square', <--- I can REMOVE this as it is the default
} as const;

export type ImageShape = ValuesOf<typeof ImageShape>;
`

Reply:
All of the values should exist in the const object - in the case we were to change the default we'd still want a way to set "square" for instance. I think Miro's primary point is that we don't want to reflect every visual default to the DOM because there's a cost to that which we don't need to pay. If it's the default, we'll just set it in CSS without tying it to the attribute itself. Consider my Avatar PR - the default size is 32 so the width/height without setting size is 32x32. Any other styles pertaining to size are written as you'd expect:
```css
:host([size="48"]) { 
    width: 48px; 
    height: 48px; 
}
```

Reply:
So, in the case of "square", I would keep the attribute but not style based on that attribute selector?

Reply:
Yes, the attribute and the value of "square" exists but for the default value you wouldn't couple the styles to the attribute itself in that case because in that case the attribute would need to be reflected for it to take. In the case of a default, we want it there whether the attribute is present or not.